src/bench/bench_bitcoin.cpp

```diff
@@ -13,6 +13,8 @@
 
 #include <memory>
 
+const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
```

Comment:
These ` = nullptr` are not needed since it's not a pointer and the empty constructor has the same behaviour.